### ISSUE TITLE
webdevops/php-nginx-dev:7.3 manifest not found

### ISSUE THREAD

**Issue by ckrack** [webdevops/Dockerfile#279]
```bash 
$ docker pull webdevops/php-nginx-dev:7.3
Error response from daemon: manifest for webdevops/php-nginx-dev:7.3 not found
```
It works with both `webdevops/php-nginx-dev:7.2` and `webdevops/php-nginx:7.3`

**Comment by hhoechtl** [comment]
Der are currently no images for `7.3-dev` because there's no xdebug version for 7.3.

**Comment by ckrack** [comment]
Ah, too bad. I also saw the commit that indicates this here.
The 7.3 tag ist still mentioned on the hub docs though: https://dockerfile.readthedocs.io/en/latest/content/DockerImages/dockerfiles/php-nginx-dev.html

**Comment by hhoechtl** [comment]
Yeah, copy paste preparation work 😅 . I'm sure 7.3 will soon be fully released with all packages

**Comment by marcusirgens** [comment]
@hhoechtl Any progress on this? Xdebug 2.7.0beta1 seems to be php 7.3 compatible.

**Comment by hhoechtl** [comment]
Last time I looked, they hadn't pushed `php:7.3-fpm` which we need as base. But the build pipeline is broken currently and I don't have time to investigate before christmas

**Comment by hhoechtl** [comment]
We now have working builds for:
* `webdevops/php:7.3`
* `webdevops/php-dev:7.3`
* `webdevops/php-apache:7.3`
* `webdevops/php-apache-dev:7.3`
* `webdevops/php-nginx:7.3`
* `webdevops/php-nginx-dev:7.3`

The `-dev` images currently don't have blackfire as it's not yet available for 7.3 but everything else is working.
